bundles/org.openhab.binding.chatgpt/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -67,7 +138,23 @@
 			<parameter name="maxTokens" type="decimal">
 				<label>Maximum Number of Tokens</label>
 				<description>The maximum number of tokens to generate in the completion.</description>
-				<default>500</default>
+				<default>1000</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="topP" type="decimal" min="0" max="1">
+				<label>TopP</label>
+
+				<description>An alternative to sampling with temperature, called nucleus sampling, where the model considers the
+					results of the tokens with top_p probability mass. So 0.1 means only the tokens comprising the top 10% probability
+					mass are considered. We generally recommend altering this or temperature but not both.</description>
+
+				<default>1</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="type" type="text">
```

Comment:
I think to remember that you said that this parameter is obsolete and can be removed?

Reply:
No, I didn't. I was talking about another parameter that has already been removed. I find the 'Top P' parameter useful.